### PR TITLE
DCC 5201 Filters on Dictionary Viewer

### PR DESCRIPTION
**Pull request by SidSyrus**
Update to jQuery 1.11

### CODE REVIEW

**Comment by btiernay** [comment]
What's the problem?

**Comment by btiernay** [comment]
@SidSyrus What is the problem?

**Comment by SidSyrus** [comment]
Its in the dcc-submission project. I've commented there for you to review.

File: theme/base.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
             ga('send', 'pageview');
         </script>
         {% endif %}
-        <script src="/js/jquery-1.10.2.min.js"></script>
+        <script src="/js/jquery-1.11.3.min.js"></script>
```
Comment (btiernay) [inline theme/base.html:58]: Shouldn't we be using npm for this?

Comment (btiernay) [reply theme/base.html:58]: @cheapsteak Can we use npm for this?

**Pull request merged**